src/tests/activity.test.js

```diff
@@ -1 +1,141 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import activity from '../models/activity';
+import activitySeed from '../seeds/activity';
+
+const mockActivity = {
+  name: 'Boxing',
+  description: 'vestibulum aliquet ultrices erat tortor sollicitudin mi non mi integer ac neque duis',
+  isActive: false,
+};
+
+const mockActivityExistingName = {
+  name: 'Chandal',
+  description: 'Any description',
+  isActive: false,
+};
+
+const mockLessName = {
+  name: '',
+  description: 'Some text',
+  isActive: true,
+};
+
+const mockLessDescription = {
+  name: 'Name',
+  description: '',
+  isActive: true,
+};
+
+const mockLessIsActive = {
+  name: 'Names',
+  description: 'some description',
+  isActive: '',
+};
+
+beforeAll(async () => {
+  await activity.collection.insertMany(activitySeed);
+});
+
+describe('GET /api/activities', () => {
+  test('The database must return 200 if is not empty of activities', async () => {
+    const response = await request(app).get('/api/activities').send();
+    expect(response.body.data.length).toBeGreaterThan(0);
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
+  test('The status must be 404, if the url is wrong', async () => {
+    const response = await request(app).get('/activities').send();
+    expect(response.status).toBe(404);
+    expect(response.error).toBeTruthy();
+  });
+  test('If are activities, data must said "Complete activities list" ', async () => {
+    const response = await request(app).get('/api/activities').send();
+    expect(response.body.message).toMatch(/Complete activities list/);
+    expect(response.error).toBeFalsy();
+  });
+});
+
+describe('GET BY ID /api/activities/:id', () => {
+  test('The database must return 200 if the id exist', async () => {
+    const response = await request(app).get('/api/activities/64677fdefc13ae39f1753b24').send();
```

Comment:
I will change this to a constant that save the id from seed, so is not manual putting the number

Reply:
Here i try to use a function to get the first id from seed, but the id is called _id, and eslint doesn't allow using a '_'. How can i solve it?


My code is:
`let firstActivityId;

beforeAll(async () => {
  await activity.collection.insertMany(activitySeed);
  const foundFirstId = await activity.find();
  if (foundFirstId.length > 0) {
    firstActivityId = foundFirstId[0]._id;
  }
});`

Reply:
I foun that i can use // eslint-disable-next-line no-underscore-dangle  but i'm not sure if it's okay or maybe doing all the logic in other way

Reply:
It's ok to use that ESLint rule.